Omer dashboard&stuff

Omer794 opened the pull request:
as i was writing below

Omer794 committed:
Dashboard,
as well added public Employee to some of the controllers.
- fixed employee login as well.
will need to make it in the center and more beauty details.

Omer794 committed:
Dashboard,
as well added public Employee to some of the controllers.
- fixed employee login as well.
will need to make it in the center and more beauty details.

Omer794 committed:
Dashboard,
as well added public Employee to some of the controllers.
- fixed employee login as well.
will need to make it in the center and more beauty details.

MacDucky commented on customerClient/src/main/java/org/lilachshop/customerclient/App.java:
```diff
@@ -29,7 +29,7 @@ public void init() throws Exception {
     @Override
     public void start(Stage stage) throws IOException {
-        scene = new Scene(loadFXML("main"));
+        scene = new Scene(loadFXML("MessageBoxComplaint"));
```
Please do not push flow dependent changes, in this case this this was probably not wanted in master.

MacDucky commented on employeeClient/pom.xml:
```diff
@@ -81,6 +81,18 @@
+            <artifactId>utilities</artifactId>
+            <version>1.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
```
why were these changes necessary?

MacDucky commented on LilachShop.iml:
```diff
@@ -8,7 +8,5 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" scope="TEST" name="Maven: junit:junit:4.11" level="project" />
-    <orderEntry type="library" scope="TEST" name="Maven: org.hamcrest:hamcrest-core:1.3" level="project" />
   </component>
 </module>
```
Please do not add iml files to git

MacDucky commented on employeeClient/src/main/java/org/lilachshop/employeeclient/CatalogEditTableController.java:
```diff
@@ -5,10 +5,12 @@
 import org.lilachshop.entities.Store;
 
 public class CatalogEditTableController {
+    public Employee employee; //   Omer added that line, as this class needs to know which employee is "using the controller"
```
this is bad, just keep a static  public employee in the dashboard, why would you pass employee to every controller...

MacDucky commented on employeeClient/src/main/java/org/lilachshop/employeeclient/ComplaintReportController.java:
```diff
@@ -24,6 +25,7 @@
 
 // todo: solve all the errors when changing stores and dates
 public class ComplaintReportController implements Initializable {
+    public Employee employee;
```
same as previous comment.

MacDucky commented on employeeClient/src/main/java/org/lilachshop/employeeclient/CustomerServiceViewController.java:
```diff
@@ -30,6 +31,8 @@
 
 
 public class CustomerServiceViewController implements Initializable {
+    public Employee employee;
+
```
same as previous comments

MacDucky commented on employeeClient/src/main/java/org/lilachshop/employeeclient/CustomerTableController.java:
```diff
@@ -8,9 +8,12 @@
 
 public class CustomerTableController {
 
+    public Employee employee;
+
```
same as previous comments.

MacDucky commented on employeeClient/src/main/java/org/lilachshop/employeeclient/EmployeeTableController.java:
```diff
@@ -35,6 +35,7 @@
 
 public class EmployeeTableController implements Initializable {
 
+    public Employee employee; // Omer added that
```
same as previous comments.

MacDucky commented on employeeClient/src/main/java/org/lilachshop/employeeclient/IncomeReportController.java:
```diff
@@ -24,6 +26,8 @@
 import java.util.ResourceBundle;
 
 public class IncomeReportController implements Initializable {
+    public Employee employee;
+
```
same as previous comments.

MacDucky commented on employeeClient/src/main/java/org/lilachshop/employeeclient/OrderReportController.java:
```diff
@@ -27,6 +28,7 @@
 import java.util.ResourceBundle;
 
 public class OrderReportController implements Initializable {
+    public Employee employee;
```
same as previous comments.

MacDucky commented on server/src/main/java/org/lilachshop/server/LilachServer.java:
```diff
@@ -24,12 +24,13 @@ public LilachServer(Integer... port) {
             throw e;
         }
-//        entityFactory.fillDataBase();
+        entityFactory.fillDataBase();
```
this is commented out on purpose

MacDucky commented on server/src/main/java/org/lilachshop/server/LilachServer.java:
```diff
@@ -24,12 +24,13 @@ public LilachServer(Integer... port) {
     @Override
     protected void handleMessageFromClient(Object msg, ConnectionToClient client) {
         // todo: switch to a request class classifying.
+        System.out.println("got here");
```
please check what you are pushing prior to pushing

MacDucky reviewed (changes requested):
Please remove the iml files, and fix the code according to what was commented.

MacDucky committed:
finished employee table.

LizaSivashova committed:
~Include touch-up to Sign up
~Display Catalog by store
~add item to Catalog
~Added attributes of color and type with Hebrew implement.
~ Refresh Table automatically

MacDucky committed:
reverting omer's artifacts

MacDucky committed:
reverting artifacts2

MacDucky committed:
last changes before merge

MacDucky committed:
Merge branch 'master' into OmerDashboard&Stuff
